config/boards/shields/totem/totem_right.overlay

```diff
@@ -11,6 +11,8 @@
 };
 
 &kscan0 {
+    // overridden here because the diodes were installed backward on this half
+    diode-direction = "row2col"; 
```

Comment:
Default is row2col, according to the documentation, so shouldn't it be col2row? https://zmk.dev/docs/config/kscan#devicetree-3

Reply:
correct, but it's [being overridden in the main overlay](https://github.com/jeroenpasman/zmk-config-totem/blob/83251955e2e2777bf82266922f688afddf1d7870/config/boards/shields/totem/totem.dtsi#L36) and we have to override it back in the right hand